README.md

```diff
@@ -18,40 +18,99 @@ Some of these are tricky!  Don't trust your first instinct.
 
 **a)** `"1" == 1`
 
+True - Due to the == not looking for an exact similarity, but a 'truthiness'.
+Console says: True
+
 **b)** `"1" === 1`
 
+False - 3 ==='s are looking for a 100% match, and string 1 is not exactly the same as number 1.
+Console says: False
+
 **c)** `x == 'x'`
```

Comment:
1c) Autoconversion doesn't happen with variables. Instead, JavaScript will look up what the variable is referring to. Here, it will only be true if `var x = "x";`.